src/sim/include/AbstractDriver.h

```diff
@@ -1240,26 +1243,34 @@ namespace MFM
             OString512 buff;
             buff.Reset();
 
-	    //for some reason, grid.GetAtomInSite doesn't work. Maybe because it's constant? Optimizing happening?
+	    //for some reason, grid.GetAtomInSite doesn't do what we need. Maybe because it's constant? Optimizing happening?
+	    //todo: figure out why that doesn't work. GetAtomInSite has an option to get the base layer, which would
+	    //be nice to have. 
             T* atom = grid.GetWritableAtom(siteInGrid);
```

Comment:
Yeah I'd think 'const T* atom = grid.getAtomInSite(getFromBase, siteInGrid);'  What goes wrong?

Reply:
I'll check this out tonight. I made that change initially back in November, so it escapes me what happens in that case. I'd love to have another option to export the base layer or append it in a separate list in the same document.

Reply:
I didn't get a chance to check on this tonight after all, but I'll check it out tomorrow.

Reply:
Finally got to check this out. You're right: GetAtomInSite works as expected. Latest commit adds base layer to JSON output!